src/main/java/com/parasoft/demoapp/dto/BuildInfoDTO.java

```diff
@@ -2,7 +2,7 @@
 
 import lombok.Data;
 
-@Data
+@Data   // parasoft-suppress PB.NUM.CLP-2 "expected"
```

Comment:
Please read the documentation to figure out how to fix it instead of suppress it. Also for "PageInfo.java". 
From the description it looks like precision of the value will be lost if we don't fix it.